src/BecklynMonitoringBundle.php

```diff
@@ -28,15 +30,23 @@ public function __construct ()
     /**
      * @inheritDoc
      */
-    public function build (ContainerBuilder $container)
+    public function build (ContainerBuilder $container) : void
     {
-        $container->addCompilerPass(
-            new RegisterAssetNamespacesCompilerPass([
-                "monitoring" => __DIR__ . "/../build",
-            ])
-        );
-
-        $container->addCompilerPass($this->releaseVersionPass);
+        if (\class_exists(RegisterAssetNamespacesCompilerPass::class))
+        {
+            $container->addCompilerPass(
+                new RegisterAssetNamespacesCompilerPass([
+                    "monitoring" => __DIR__ . "/../build",
+                ]
+                )
+            );
```

Comment:
Weird CS

Reply:
Ah yeah, that was PhpStorm with it's glorious reformatting after adding a `)` somewhere around it.